apps/admin-gui/src/app/facilities/pages/resource-detail-page/resource-detail-page.component.html

```diff
@@ -9,6 +9,20 @@ <h1 class="page-title">
     <span class="text-muted">
       &nbsp;#{{resource.id}}
     </span>
+    <br>
+    <span class="text-muted mt-1 entity-info">
+      {{'RESOURCE_DETAIL.VO' | translate}}:
+      <span class="resource-text">
+        {{resource.voId}} / {{resource.vo.name}}
```

Comment:
I would like the voId by represented as &nbsp;#{{resource.voId}} and after the name of the vo